packages/core/src/components/menu-avatar/menu-avatar.tsx

```diff
@@ -60,6 +73,10 @@ export class MenuAvatar {
 
   private avatarElementId = 'ix-menu-avatar-id';
 
+  componentWillLoad() {
+    this.hasContent = this.hostElement.innerHTML !== '';
+  }
+
```

Comment:
This will on work once the component is loaded.
Following code example show that behavior:

```
@State() toggle = false;

  render() {
    return (
      <Host>
        <ix-application>
          <ix-menu>
            <ix-menu-avatar showLogoutButton={false}>
              {this.toggle && <ix-dropdown-item>Test</ix-dropdown-item>}
            </ix-menu-avatar>
          </ix-menu>
          <ix-button onClick={() => (this.toggle = !this.toggle)}>
            test
          </ix-button>
        </ix-application>
      </Host>
    );
  }
```

Maybe a implementation like this one https://github.com/siemens/ix/blob/e4fe29d2995e15bf5869c290081a5c3df8444213/packages/core/src/components/content/content.tsx#L41 will also fit for this usecase?